### DISCUSSION TITLE
Is it possible to just reduce the bitrate of a video without changing anything else about it?

### DISCUSSION THREAD

**Discussion by HERIB0T** [mifi/lossless-cut#2082] [Q&A]
Firstly sorry for my English, I'm using Google Translate for this.
My question is whether LLC is capable of just reducing the bit rate of a video without changing any other information in the file. Many programs do this, but they re-encode the file and it takes many hours to do so. I am grateful in advance.

**Comment by mifi** [comment]
you can only change the bitrate by discarding unneeded tracks

**Reply by HERIB0T** [reply]
So it's not possible, right? why discarding unnecessary tracks does not change the bit rate, but rather the time of the video.

**Reply by mifi** [reply]
if you have a video with an audio track and a video track, and you discard the audio track, it should reduce the overall bitrate of your file.

**Reply by HERIB0T** [reply]
So is there only this way? I just wanted to reduce the bitrate of the video track.

**Reply by mifi** [reply]
as far as i know it's the only way without re-encoding. if you want to re-encode the vidoe track, then you should use a different tool than losslesscut (because it's a lossy operation)
